### PR TITLE
8271142: package help is not displayed for missing X11/extensions/Xrandr.h

### PR DESCRIPTION
**Pull request by lusou-zhangquan**
Compared with the origin commit in JDK-17, this backport is no clean for two reasons:
1. JDK11 doesn't have package handler `zypper`
2. JDK11 has the extra package handler `apk`

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] [JDK-8271142](https://bugs.openjdk.org/browse/JDK-8271142) needs maintainer approval

### Issue
 * [JDK-8271142](https://bugs.openjdk.org/browse/JDK-8271142): package help is not displayed for missing X11/extensions/Xrandr.h (**Bug** - P4 - Approved)

### Reviewers
 * [Christoph Langer](https://openjdk.org/census#clanger) (@RealCLanger - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk11u-dev.git pull/2538/head:pull/2538` \
`$ git checkout pull/2538`

Update a local copy of the PR: \
`$ git checkout pull/2538` \
`$ git pull https://git.openjdk.org/jdk11u-dev.git pull/2538/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 2538`

View PR using the GUI difftool: \
`$ git pr show -t 2538`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk11u-dev/pull/2538.diff">https://git.openjdk.org/jdk11u-dev/pull/2538.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk11u-dev/pull/2538#issuecomment-1958590545)

### CODE REVIEW

**Comment by lusou-zhangquan** [comment]
@RealCLanger Could you please take a look at this PR? I port JDK-9271142 from JDK17 because the same problem happens when building JDK11 and this patch helps a lot.

File: make/autoconf/help.m4 [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2011, 2022, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2011, 2024, Oracle and/or its affiliates. All rights reserved.
```
Comment (RealCLanger) [inline make/autoconf/help.m4:2]: Please revert the copyright change to keep the original (2011, 2022).

**Review by RealCLanger — CHANGES_REQUESTED** [review]

File: make/autoconf/help.m4 (hunk 1 above)
Comment (lusou-zhangquan) [reply make/autoconf/help.m4]: OK, I've reverted the copyright change.

**Review by RealCLanger — APPROVED** [review]
Looks good now. Please request approval.

**Comment by lusou-zhangquan** [comment]
/approval request Backport to JDK11, not clean for two reasons: 1. JDK11 doesn't have package handler zypper
2. JDK11 has the extra package handler apk

**Comment by lusou-zhangquan** [comment]
> @lusou-zhangquan, next time please tell how you tested your change in the approval comment!

Before creating this PR, I've tested this commit locally and package help messages was printed clearly. Sorry for the absent of test messages in approval comment, won't make the same mistake next time.

**Pull request closed without merging**